plugins/header_rewrite/parser.cc

```diff
@@ -50,6 +52,12 @@ Parser::Parser(const std::string &line) : _cond(false), _empty(false)
         _tokens.push_back(std::string(1, line[i]));
       }
       continue; /* always eat whitespace */
+    } else if (line[i] == '\\') {
+      // erase a backslash in quoted-string
+      if (inquote && extracting_token) {
```

Comment:
Why the ``inquote`` condition? Why can't backquotes work outside of a quoted string?